src/qt/balancesdialog.h

```diff
@@ -58,24 +46,22 @@ class BalancesDialog : public QDialog
     QMenu *contextMenuSummary;
 
     GUIUtil::TableViewLastColumnResizingFixer *borrowedColumnResizingFixer;
-    virtual void resizeEvent(QResizeEvent* event);
+    virtual void resizeEvent(QResizeEvent *event);
```

Comment:
You're right, and *actually* I prefer `X* variable` over `X *variable`, *but* in this case all others used `X *variable`, too. :)